File: tests/fixtures/good.py
#!/usr/bin/env python3
# Copyright 2026 René El Fatuaki
# Licensed under the MIT License
# https://opensource.org/license/mit
import os

def parse_cursor(raw: str) -> str:
    """Return the cursor encoded in the pagination token."""
    return raw

async def fetch_user(user_id: int) -> int:
    """Fetch the user with a per-request timeout instead of the session default.

    `aiohttp client timeouts <https://docs.aiohttp.org/en/stable/client_quickstart.html#timeouts>`_
    """
    # Resolve the token before the first request.
    # https://docs.aiohttp.org/en/stable/client_quickstart.html
    token = user_id  # noqa: F841
    return token

# TODO: handle timeout https://github.com/example/app/issues/7
def todo(): pass

File: tests/fixtures/good.swift
// Copyright 2026 René El Fatuaki
// Licensed under the MIT License
// https://opensource.org/license/mit
import Foundation

// MARK: - Lifecycle
/// Fetches the user and caches the result for the session.
/// [URLSession](https://developer.apple.com/documentation/foundation/urlsession)
func fetchUser() {}

// TODO: retry on timeout https://github.com/example/app/issues/42
func retry() {
    // Resolve the token before the first request.
    // https://developer.apple.com/documentation/security/keychain-services
    let token = 1
    // Apply the changes atomically.
    _ = token
}

/// Uses `a; b` inline code and the &amp; entity.
func inlineCode() {}

let multi = """
  a — b; // still a string
  """
